php-checks/src/main/java/org/sonar/php/checks/TooManyFunctionParametersCheck.java

```diff
@@ -54,12 +59,19 @@
   public void visitParameterList(ParameterListTree parameterList) {
     int numberOfParameters = parameterList.parameters().size();
     int maxValue = isConstructorParameterList(parameterList) ? constructorMax : max;
-    if (numberOfParameters > maxValue) {
+    if (numberOfParameters > maxValue && isOverriding(parameterList.getParent()).isFalse()) {
```

Comment:
Should we raise an issue when it is unknown whether a method is overriding?
```suggestion
    if (numberOfParameters > maxValue && !isOverriding(parameterList.getParent()).isTrue()) {
```